client/src/components/Main/Main.jsx

```diff
@@ -11,20 +9,24 @@ import SurveyDetail from '../../pages/SurveyDetail';
 import TakeSurvey from '../../pages/TakeSurvey';
 
 const Main = () => {
-  // const { auth } = useSelector((state) => state.adminLoginReducer);
-  // const snackbar = useSelector((state) => state.snackbarReducer);
+  const { data } = useSelector((state) => state.adminLoginReducer);
+  const { auth } = data;
+  const { snackbar } = useSelector((state) => state.snackbarReducer);
```

Comment:
storing React elements in the state isn't good practice unforunately, but your solution is v elegant.

Reply:
Did not know it was bad practice! 
Issue #35 added.